README.md

```diff
@@ -292,14 +308,15 @@ Provide multiple paths/patterns:
 
 ```js
 {
-  title: 'Multiple Files',  
+  title: 'Multiple Files',
   files: [
     'my/project/styles/global/*',
     'my/project/styles/components/feedback.scss',
     '!my/project/styles/global/colors.scss'
   ]
 }
 ```
+*`!` negates the match*
```

Comment:
I hesitated with that too. I don't think we should be including this line. We reference Globby and give a link to it on line 285, I think that should be enough. Globby has their own documentation, we shouldn't be re-iterating it. Do you think we need to include lines 321-331 then as well?